File: source/csharp/glympse/hal/GBatteryListener.cs
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

using System;
using System.Collections;
using System.Collections.Generic;
using System.Text;
using System.Threading;

namespace Glympse 
{
    
/*J*public**/ interface GBatteryListener : GCommon
{    
     void updateStatus(int level, bool plugged, bool present);

     void memoryWarningReceived();
};
    
/*C*typedef GBatteryListener  GBatteryListener;**/
    
}

File: source/java/com/glympse/java/hal/GBatteryListener.java
//------------------------------------------------------------------------------
//
// Copyright (c) 2014 Glympse Inc.  All rights reserved.
//
//------------------------------------------------------------------------------

package com.glympse.java.hal;

import java.util.*;
import com.glympse.java.core.*;
import com.glympse.java.hal.*;
    
/*J*/public/**/ interface GBatteryListener extends GCommon
{    
    public void updateStatus(int level, boolean plugged, boolean present);

    public void memoryWarningReceived();
};
    
/*C*typedef GBatteryListener  GBatteryListener;**/
    


